suite-native/module-settings/src/components/AboutUsBanners.tsx

```diff
@@ -35,41 +36,39 @@ export const AboutUsBanners = () => {
                         variant="titleSmall"
                         style={applyStyle(trezorDescriptionTextStyle)}
                     >
-                        Trezor Suite Lite is a safe and secure way to stay connected to the crypto
-                        on your hardware wallet. Track coin balances on the go without exposing your
-                        private data. Easily create and send payment addresses to anyone.
+                        <Translation id="moduleSettings.aboutUs.body" />
                     </Text>
                 </VStack>
             </Card>
             <Card style={applyStyle(cardStyle, { backgroundColor: 'backgroundSecondaryDefault' })}>
                 <VStack spacing="sp24" style={applyStyle(stackStyle)}>
                     <Text color="textDefaultInverted" variant="titleMedium">
-                        Follow us
+                        <Translation id="moduleSettings.aboutUs.followUs" />
                     </Text>
                     <HStack spacing="sp24">
                         <IconButton
                             size="large"
                             colorScheme="tertiaryElevation1"
-                            iconName="facebookLogo"
+                            iconName="twitterLogo"
                             accessibilityRole="link"
-                            accessibilityLabel="facebook"
-                            onPress={() => openLink('https://www.facebook.com/trezor.io')}
+                            accessibilityLabel="X"
+                            onPress={() => openLink('https://x.com/trezor')}
```

Comment:
I can't open X or tiktok if I have them installed on Android phone 😮 
<img width="374" alt="image" src="https://github.com/user-attachments/assets/5a42b591-785e-4ce4-85f0-8269f73b54f6" />

Reply:
@PeKne @matejkriz **X** and **Instagram** worked as expected for me, but **TikTok** did not, try to change the url to `https://tiktok.com/@trezor.io_official `(removing *www** part) please https://github.com/trezor/trezor-suite/pull/17086/commits/86ca7849695b9cd1a3582b0d5fb88018b03dc60e



https://github.com/user-attachments/assets/90d5d6fa-8ee3-4fb7-8ee1-e5534ae49b79